console/src/views/ThemeSettings.js

```diff
@@ -227,14 +238,29 @@ ThemeSettingsBase.propTypes = {
 
 ThemeSettingsBase.displayName = 'ThemeSettings';
 
+const ColorCheckboxItem = kind({
```

Comment:
Please move this at the top pf the file where the other custom components are defined. 
Also move DynamicColorSetting below, after SkinVariantsSetting.